stripe/.idea/libraries/Dart_SDK.xml

```diff
@@ -1,19 +0,0 @@
-<component name="libraryTable">
-  <library name="Dart SDK">
-    <CLASSES>
-      <root url="file:///Users/jaimeblasco/Documents/Projects/flutter/bin/cache/dart-sdk/lib/async" />
-      <root url="file:///Users/jaimeblasco/Documents/Projects/flutter/bin/cache/dart-sdk/lib/collection" />
-      <root url="file:///Users/jaimeblasco/Documents/Projects/flutter/bin/cache/dart-sdk/lib/convert" />
-      <root url="file:///Users/jaimeblasco/Documents/Projects/flutter/bin/cache/dart-sdk/lib/core" />
-      <root url="file:///Users/jaimeblasco/Documents/Projects/flutter/bin/cache/dart-sdk/lib/developer" />
-      <root url="file:///Users/jaimeblasco/Documents/Projects/flutter/bin/cache/dart-sdk/lib/html" />
-      <root url="file:///Users/jaimeblasco/Documents/Projects/flutter/bin/cache/dart-sdk/lib/io" />
-      <root url="file:///Users/jaimeblasco/Documents/Projects/flutter/bin/cache/dart-sdk/lib/isolate" />
-      <root url="file:///Users/jaimeblasco/Documents/Projects/flutter/bin/cache/dart-sdk/lib/math" />
-      <root url="file:///Users/jaimeblasco/Documents/Projects/flutter/bin/cache/dart-sdk/lib/mirrors" />
-      <root url="file:///Users/jaimeblasco/Documents/Projects/flutter/bin/cache/dart-sdk/lib/typed_data" />
-    </CLASSES>
-    <JAVADOC />
-    <SOURCES />
-  </library>
-</component>
```

Comment:
Just deleting all .idea folder from the Git since seems like previously this was not gitignored properly!